content-moderation-evaluation/Evaluating-Amazon-Rekognition-Content-Moderation-Service.ipynb

```diff
@@ -0,0 +1,542 @@
+    "with open(\"class_labels.json\", \"w\") as f:\n",
+    "    json.dump(json_body, f)\n",
+    "\n",
+    "s3.upload_file(\"class_labels.json\", BUCKET, EXP_NAME + \"/class_labels.json\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Create instruction template for Ground Truth Workforce\n",
+    "All of your evaluation dataset will be annotated by human annotators. It is critical to provide clear and concise instructions that help the annotators understand what you want to achieve. When used through the AWS Console, Ground Truth helps you create the instructions using a visual wizard. When using the API, you need to create an HTML template for your instructions. Below, we prepare a very simple but effective template and upload it to your S3 bucket.\n",
+    "\n",
+    "_NOTE:_\n",
+    "- _If you use any images in your template (as we do), they need to be publicly accessible. You can enable public access to files in your S3 bucket through the S3 Console, as described in [3 Documentation](https://aws.amazon.com/premiumsupport/knowledge-center/read-access-objects-s3-bucket/). In this case, we need to have X+1 number of images, X equals to number of moderation labels you like to use plus \"safe content\" label. In our example, it's 6._ \n",
+    "- _Replace placements with the actual image file names and moderation labels. e.g. image_1, image_2, ..., image_n, moderation_label_1, moderation_label_2, ..., moderation_label_n_\n",
+    "- _The sample images image_1, image_2, etc., corresponds to moderation_label_1, moderation_label_2, etc., mentioned above, and image_6 should be an example of safe content._\n",
+    "- _If you want to change number of moderation labels, you need to adjust sample images and labels accordingly._"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Testing your template\n",
+    "It is very easy to create broken template, This might not only cause your labeling job to fail but also cause your job to complete with meaningless results (when the annotators have no idea what to do, or the instructions are plain wrong). We highly recommend that you verify that your task is correct. The following cell creates and uploads a file called instructions.template to S3. It also creates instructions.html that you can open in a local browser window. Please do so and inspect the resulting web page; it should correspond to what you want your annotators to see (except the actual image to annotate will not be visible)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create the ground truth instruction template\n",
+    "# make sure those objects are pubicly accessible in s3 bucket\n",
+    "img_examples = [\n",
+    "    \"https://s3.amazonaws.com/contentmoderation-accuracy-dataset/images/{}\".format(img_id)\n",
```

Comment:
I think we discussed this but I probably misunderstood. Are we providing these images to customers in the `contentmoderation-accuracy-dataset` or not? You are implying this needs to be public but if this is in the customer account, that is an action on us, why are you asking them to make it public?

I think that
1. If this is our bucket: a) it doesn't seem to be public to me right now b) remove `make sure those objects are publicly accessible in s3 bucket` line c) verify paths because the don't have a file extension such as `.jpg`
2. If this is not our bucket: a) replace `contentmoderation-accuracy-dataset` with `$BUCKET` or just use `filenames` from previous cells b) Why public? Can we specify instructions to setup bucket policy instead?